scripts/mysqldumpslow.sh

```diff
@@ -1,13 +1,7 @@
 #!@PERL_PATH@
 
-<<<<<<< HEAD
```

Comment:
Likewise

Reply:
I see what you mean. I guess it is related with the way git carries out the [merging of files](https://softwareengineering.stackexchange.com/questions/194788/why-doesnt-git-merge-adjacent-lines-without-conflict/378258#378258).

Although in this particular case, it seems we didn't merge the merge-base change at the time.